frontend/src/components/EditProfile/EditProfile.scss

```diff
@@ -0,0 +1,125 @@
+.edit-profile-form label {
+  display: inline-block;
+  margin: 0 50px;
+  clear: left;
+  text-align: right;
+  width: 100px;
+}
+
+.edit-profile-form {
+  width: 1000px;
+}
+
+.edit-profile-button {
+  width: 100px;
+  padding-left: 420px;
+  margin-top: 10px;
+}
+.btn {
+  color: rgb(25, 136, 240);
+  background-color: rgb(40, 40, 40);
+  outline: none;
+  border: none;
+  padding-left: 20px;
+  padding-bottom: 10px;
+  padding-top: 10px;
+  text-transform: uppercase;
+  border-radius: 20px;
+  cursor: pointer;
+  display: inline-block;
+  text-align: right;
+}
+.edit-profile {
```

Comment:
With SCSS, you can nest styling to make for better legibility (if not overused) so in this case, the styles relating to any .edit-profile element can be nested like you did for `nav ul`